src/v3/src/components/InfoBox/InfoBox.tsx

```diff
@@ -56,7 +56,14 @@ const InfoBox: UISchemaElementComponent<{
         {...({ 'data-se': dataSe })}
         className={`infobox-${messageClass.toLowerCase()}`}
       >
-        { title && <AlertTitle>{title}</AlertTitle> }
+        { title && (
+        <AlertTitle
+          role="heading"
+          aria-level="3"
```

Comment:
Since v2 uses an `<h3>` header tag for its error title, this would match parity for accessibility and testing.  Correct me if im wrong

Reply:
In case anyone is curious where these allowed props are coming from like I was, `AlertTitle` is a styled version of the `Typography` component, so can be passed any properties that are allowed on `Typography`. The MUI documentation doesn't make that clear at all so I had to dig into the source code to find that 🙂

Reply:
Does this work if we apply `component` and `variant='h3'` instead? If so it should come with free implied role/aria-level attributes.

Reply:
> In case anyone is curious where these allowed props are coming from like I was, `AlertTitle` is a styled version of the `Typography` component, so can be passed any properties that are allowed on `Typography`. The MUI documentation doesn't make that clear at all so I had to dig into the source code to find that 🙂

awesome didnt know that thanks for that!

Reply:
> 

`component` and `variant='h3'` dont seem to be valid for `<AlertTitle>` and I can't find anything in the docs that says otherwise

Reply:
I think maybe because we are using the `<AlertTitle>` from `@okta/odyssey-react-mui`?

Reply:
As far as I can tell the `AlertTitle` in `@okta/odyssey-react-mui` is just directly reexported from `@mui/material`

Reply:
Oh, it doesn't look like the MUI material AlertTitle passes all the props through to the inner Typography component :/
https://github.com/mui/material-ui/blob/5d387c0e588c163a0c8c159ab53243b9d895a10b/packages/mui-material/src/AlertTitle/AlertTitle.js#L20-L52